### PR TITLE
Update gas configs and chain_info

### PR DESCRIPTION
**Pull request by codebycarson**
Updated chain_info.json to be pacific-1 and updated gas configs to the correct price

### CODE REVIEW

**Comment by LCyson** [comment]
can we prob follow how other chains structure this repo: https://github.com/osmosis-labs/assetlists? we prob want to check if this break fin/compass atlatnic-2 integration

**Comment by codebycarson** [comment]
> can we prob follow how other chains structure this repo: https://github.com/osmosis-labs/assetlists? we prob want to check if this break fin/compass atlatnic-2 integration

We went with the standard cosmos asset list structure. https://github.com/cosmos/chain-registry

We don't have all the info, but we picked the parts we needed and kept the schema the same to help the wallet providers who are cross chain cosmos wallets

Same type of schema used in the keplr wallet configs.
https://github.com/chainapsis/keplr-chain-registry/blob/main/cosmos/osmosis.json

File: chain_info.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "chain_name": "Sei Atlantic 2",
-  "network_type": "testnet",
-  "chain_id": "atlantic-2",
+  "chain_name": "Sei",
```
Comment (besated) [inline chain_info.json:2]: should we a list of chains in here? so we can have multiple chain info for atlantic-2 and pacific-1?

Comment (codebycarson) [reply chain_info.json:2]: We have the different chain info in the chains.json, I believe this is simply redundant info we don't need.

**Review by besated — APPROVED** [review]

**Pull request merged**